spec/integration/commands/activedocs_command/apply_command_spec.rb

```diff
@@ -0,0 +1,85 @@
+RSpec.describe 'ActiveDocs Apply command' do
+  include_context :real_api3scale_client
+  include_context :random_name
+  include_context :resources
+  subject { ThreeScaleToolbox::CLI.run(command_line_str.split) }
+  let(:remote) do
```

Comment:
`let(:remote) { client_url }` should work

Reply:
:+1: